.wvr/build-config.js

```diff
@@ -0,0 +1,156 @@
+const config = {
+  path: './dist',
+
+  // Webpack dev server public path.
+  publicPath: '/status',
+
+  terserOptions: {
+    ecma: 5,
+    format: {},
+    parse: {},
+    compress: {
+      unused: false,
+    },
+    mangle: false,
+    module: false
+  },
+
+  copy: [
+    {
+      from: './node_modules/bootstrap/fonts',
+      to: './resources/styles/bootstrap/dist/fonts'
+    },
+    {
+      from: './node_modules/@wvr/core/app/resources/images',
+      to: './node_modules/@wvr/core/app/resources/images'
+    },
+    {
+      from: './node_modules/@wvr/core/app/views',
+      to: './node_modules/@wvr/core/app/views'
+    },
+    {
+      from: './app/index.html',
+      to: './index.html'
+    },
+    {
+      from: './build/appConfig.js.template',
+      to: './appConfig.js',
+      transform(content) {
+        return content
+          .toString()
+          .replace('${STOMP_DEBUG}', 'false')
+          .replace('${AUTH_SERVICE_URL}', 'https://labs.library.tamu.edu/authfix')
+          .replace('${WEB_SERVICE_URL}', 'http://localhost:9000')
```

Comment:
Should this include the context path for the service?
```suggestion
-          .replace('${WEB_SERVICE_URL}', 'http://localhost:9000')
+          .replace('${WEB_SERVICE_URL}', 'http://localhost:9000/status')
```

Reply:
Deferring for next phase.